data_model/src/metadata.rs

```diff
@@ -378,18 +196,21 @@ impl Metadata {
     pub fn insert_with_limits(
         &mut self,
         key: Name,
-        value: impl Into<MetadataValueBox>,
+        value: JsonString,
```

Comment:
```suggestion
        value: impl Into<JsonString>,
```

and add different implementations of `From<JsonString>`. This will look much nicer for the client

Reply:
you will be able to replace every `JsonString::new(x)` with just `x` in function calls

Reply:
> and add different implementations of `From<JsonString>`. This will look much nicer for the client

I did it before, but then I & @Erigara decided to reduce the code by generalizing via TryFrom<JsonString> for JsonCandidate<T> because serde_json may throw an error.

Reply:
implement for the ones that are infallible. For example, you can implement `From<bool> for JsonString`, you can also implement for integers